pages/api/lnurlp/[username]/pay.js

```diff
@@ -30,22 +29,23 @@ export default async ({ query: { username, amount, nostr, comment, payerdata: pa
       // If there is an amount tag, it MUST be equal to the amount query parameter
       const eventAmount = note.tags?.find(t => t[0] === 'amount')?.[1]
       if (schnorr.verify(note.sig, note.id, note.pubkey) && hasPTag && hasETag && (!eventAmount || Number(eventAmount) === Number(amount))) {
-        description = user.hideInvoiceDesc ? undefined : 'zap'
+        description = 'zap'
```

Comment:
why are we not respecting this setting anymore?

Reply:
because it is now passed to a paid action, so it is redundant since paid actions already respect this settings internally when they create their invoices:

for SN invoices:

https://github.com/stackernews/stacker.news/blob/4675a2c29d1de4b43b34e1bdda32a3f7b83ac81b/api/paidAction/index.js#L299


and p2p: 

https://github.com/stackernews/stacker.news/blob/4675a2c29d1de4b43b34e1bdda32a3f7b83ac81b/wallets/server.js#L159